src/components/Header/Header.jsx

```diff
@@ -0,0 +1,35 @@
+import styles from "@styles/Header.module.css";
+import logoImg from "@assets/images/logo.png";
+import logoTextImg from "@assets/images/logo-text.png";
+import { Link, Outlet } from "react-router-dom";
+import { useState } from "react";
+import Navigations from "./Navigations";
+import HeaderAuth from "./HeaderAuth";
+
+function Header() {
+  const [isLogined, setIsLogined] = useState(sessionStorage.getItem("logined"));
```

Comment:
```suggestion
  const [isLoggedIn, setIsLoggedIn] = useState(sessionStorage.getItem("loggedIn"));
```

문법상 isLoggedIn이 맞습니다.
간단한 단어 교정이나 오타 검증은 아래 extension 도움을 받아보세요! :)

[streetsidesoftware.code-spell-checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)